docs/pages/machine-workload-identity/getting-started.mdx

```diff
@@ -176,13 +166,16 @@ spec:
         verbs:
           - '*'
 ```
-</details>
 
-- Replace the `kubernetes_labels` selector with your own
+Replace:
```

Comment:
Should we use `<Var />`s instead of asking the user to replace stuff manually?

Reply:
Agreed - I want to address this in another PR since I think there's some interesting choices to make about whether we front-load the information collection in a Step 0 - or if they fill them in as they go through.